plugins/venat-plugin-xiv-weather/src/commands/weather.command.ts

```diff
@@ -0,0 +1,76 @@
+import { TransformPipe } from '@discord-nestjs/common';
+import {
+  Command,
+  DiscordTransformedCommand,
+  Payload,
+  TransformedCommandExecutionContext,
+  UsePipes,
+} from '@discord-nestjs/core';
+import { Logger } from '@nestjs/common';
+import { MessageEmbed } from 'discord.js';
+import { Forecast, Forecaster } from '../data/weather';
+import { WeatherDto } from '../dto/weather.dto';
+/// <reference path="colorthief.d.ts"/>
```

Comment:
need this?

Reply:
Yeah, that's used for linking the import to a declaration file, since it's a plain-JS module. It's a meta comment that TypeScript language servers can use, even though vscode's language server doesn't require it.